src/main/java/com/facebook/openwifirrm/ucentral/WifiScanEntry.java

```diff
@@ -38,23 +31,14 @@ public WifiScanEntry() {}
 	public WifiScanEntry(WifiScanEntry o) {
 		super(o);
 		this.unixTimeMs = o.unixTimeMs;
-		this.country = o.country;
-		this.qbssLoad = o.qbssLoad;
-		this.localPowerConstraint = o.localPowerConstraint;
-		this.txPwrInfo = o.txPwrInfo;
+		this.ies = o.ies; // TODO deep copy?
```

Comment:
Should I deep copy here? Either way, I'll address and delete this TODO comment

Reply:
I think shallow is fine, we're not planning to modify any fields